### ISSUE TITLE
TypeCasting issue when running execute.py in train mode

### ISSUE THREAD

**Issue by raghavSangamnerkar** [suriyadeepan/easy_seq2seq#21]
File "/home/raghav/development/machineLearning/tensorflow/_python_build/tensorflow/python/framework/op_def_library.py", line 522, in apply_op
    inferred_from[input_arg.type_attr]))
TypeError: Input 'y' of 'Mul' Op has type float32 that does not match type int32 of argument 'x'.

**Comment by canivel** [comment]
same here, did you found a solution?

**Comment by raghavSangamnerkar** [comment]
No, I was running it on ubuntu and it gave the same error no matter how many times I cleaned it and ran again. I didn't change anything, did the same thing on Mac and voila, everything worked just fine. The model did not give good results though, even after I trained it for approx. a week.
